src/state.js

```diff
@@ -40,7 +40,7 @@ var init = function(container) {
   rules = container.rules || rules;
   dataElements = container.dataElements || dataElements;
   buildInfo = container.buildInfo || buildInfo;
-  propertySettings = container.propertySettings || propertySettings;
+  propertySettings = container.property.settings || propertySettings;
```

Comment:
Considering `property` and `buildInfo` are both [required in the container](https://git.corp.adobe.com/reactor/turbine-schemas/blob/master/schemas/container.json#L235-L238), I would be okay with changing these two lines to:

```
buildInfo = container.buildInfo;
propertySettings = container.property.settings;
```

Your choice.